### ISSUE TITLE
Dumpling errors over shared files

### ISSUE THREAD

**Issue by danmosemsft** [dotnet/corefx#19086]
https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_debug/34/consoleText
```
  Executing in D:\j\workspace\outerloop_net---903ddde6\bin\Windows_NT.AnyCPU.Debug\Common.Tests\netstandard\ 
  Running tests... Start time: 15:24:25.99
  downloaded dumpling.py
  total elapsed time 0:00:00.341000
  downloaded dumpling.py
  total elapsed time 0:00:00.341000
  Traceback (most recent call last):
  Traceback (most recent call last):
    File "dumpling.py", line 1128, in <module>
    File "dumpling.py", line 1128, in <module>
      main(sys.argv)
    File "dumpling.py", line 1123, in main
      main(sys.argv)
      cmdProc.Process(config)
    File "dumpling.py", line 1123, in main
    File "dumpling.py", line 537, in Process
      self.Install(config)
    File "dumpling.py", line 545, in Install
      self._dumpSvc.DownloadClientFile('dumpling.py', config.installpath) 
    File "dumpling.py", line 213, in DownloadClientFile
      DumplingService._stream_file_from_response(response, os.path.join(downdir, filename))  
    File "dumpling.py", line 384, in _stream_file_from_response
      os.remove(path)
  WindowsError: [Error 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\dotnet-bot\\.dumpling\\dumpling.py'
      cmdProc.Process(config)
    File "dumpling.py", line 537, in Process
      self.Install(config)
    File "dumpling.py", line 545, in Install
      self._dumpSvc.DownloadClientFile('dumpling.py', config.installpath) 
    File "dumpling.py", line 213, in DownloadClientFile
      DumplingService._stream_file_from_response(response, os.path.join(downdir, filename))  
    File "dumpling.py", line 384, in _stream_file_from_response
      os.remove(path)
  WindowsError: [Error 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\dotnet-bot\\.dumpling\\dumpling.py'
  downloaded dumpling.py
  downloading debugger for client windows-amd64
  Traceback (most recent call last):
    File "C:\Users\dotnet-bot/.dumpling/dumpling.py", line 1128, in <module>
      main(sys.argv)
    File "C:\Users\dotnet-bot/.dumpling/dumpling.py", line 1123, in main
      cmdProc.Process(config)
    File "C:\Users\dotnet-bot/.dumpling/dumpling.py", line 537, in Process
      self.Install(config)
    File "C:\Users\dotnet-bot/.dumpling/dumpling.py", line 566, in Install
      dbgPath = self._dumpSvc.DownloadDebugger(dbgdir)
    File "C:\Users\dotnet-bot/.dumpling/dumpling.py", line 192, in DownloadDebugger
      DumplingService._stream_zip_archive_from_response(response, outputdir)
    File "C:\Users\dotnet-bot/.dumpling/dumpling.py", line 355, in _stream_zip_archive_from_response
      zip.extract(path, unpackdir)
    File "C:\Python27\lib\zipfile.py", line 1031, in extract
  downloaded dumpling.py
  downloaded dumpling.py
      return self._extract_member(member, path, pwd)
    File "C:\Python27\lib\zipfile.py", line 1082, in _extract_member
      os.mkdir(targetpath)
  WindowsError: [Error 183] Cannot create a file when that file already exists: 'C:\\Users\\dotnet-bot\\.dumpling\\dbg\\dbg\\triage'
  downloaded dumpling.py
  downloading debugger for client windows-amd64
  Adding debugger settings dumpling config
  configuration saved to C:\Users\dotnet-bot\.dumpling\dumpling.config.json
  Debugger successfully installed
  downloaded analysis.py
  downloaded triage.ini
```

**Comment by mellinoe** [comment]
We should fix this, but we don't need to do it for 2.0. It doesn't cause actual failures, just crappy output in the tests, and some projects won't have dumpling active if it happens. It's crappy, but not critical, IMO. It is essentially this: https://github.com/dotnet/buildtools/issues/1360.

**Comment by wtgodbe** [comment]
I don't believe we use dumpling anymore with the new CI, closing. @safern please re-open if I'm incorrect
